docs/requirements.md

```diff
@@ -22,8 +22,8 @@ The cluster running Codacy must satisfy the following requirements:
 
 -   The infrastructure hosting the cluster must be provisioned with the hardware and networking requirements described below
 -   The orchestration platform managing the cluster must be one of:
-    -   [Kubernetes](https://kubernetes.io/) **version 1.19.\*** to **1.22.\*** (1.22 recommended)
-    -   [MicroK8s](https://microk8s.io/) **version 1.19.\*** to **1.22.\*** (1.19 recommended)
+    -   [Kubernetes](https://kubernetes.io/) **version 1.19.\*** to **1.23.\*** (1.23 recommended)
+    -   [MicroK8s](https://microk8s.io/) **version 1.19.\*** to **1.23.\*** (1.22 recommended)
```

Comment:
What about using this opportunity to drop support for [older Kubernetes/MicroK8s versions](https://kubernetes.io/releases/patch-releases/#non-active-branch-history)?

Reply:
We chose to deprecate the Kubernetes version to accompany those currently [supported by EKS](https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html#kubernetes-release-calendar). Regarding MicroK8s, we prefer to retain the minimum available version at 1.19, even considering it to be deprecated upstream, since we are only now passing our recommendation to version 1.22. We wouldn't want users to rush MicroK8s version upgrades to our recommended one and break their installations in the process. The MicroK8s version gap can be narrowed in a future opportunity.